src/v3.ts

```diff
@@ -73,7 +73,7 @@ export default function generateTypesV3(
           (node.enum as string[]).map((item) =>
             typeof item === "number" || typeof item === "boolean"
               ? item
-              : `'${item}'`
+              : `'${item.replace(/'/g, "\\'")}'`
```

Comment:
This makes sense to me. In my experience, you should never be too aggressive escaping quotes like this, as you may end up with more things escaped than anticipated.